x-pack/plugins/siem/public/components/field_renderers/field_renderers.test.tsx

```diff
@@ -211,10 +213,35 @@ describe('Field Renderers', () => {
   describe('#reputationRenderer', () => {
     test('it renders correctly against snapshot', () => {
       const wrapper = shallowWithIntl(
-        <TestProviders>{whoisRenderer('10.10.10.10')}</TestProviders>
+        <TestProviders>{reputationRenderer('10.10.10.10')}</TestProviders>
       );
 
       expect(toJson(wrapper)).toMatchSnapshot();
     });
   });
+
+  describe('DefaultFieldRenderer', () => {
+    test('it should render a single item', () => {
+      const wrapper = mountWithIntl(
+        <TestProviders>
+          <DefaultFieldRenderer rowItems={['item1']} attrName={'item1'} idPrefix={'prefix-1'} />
+        </TestProviders>
+      );
+      expect(wrapper.text()).toEqual('item1 ');
+    });
+
+    test('it should render two items', () => {
+      const wrapper = mountWithIntl(
+        <TestProviders>
+          <DefaultFieldRenderer
+            displayCount={5}
+            rowItems={['item1', 'item2']}
+            attrName={'item1'}
+            idPrefix={'prefix-1'}
+          />
+        </TestProviders>
+      );
+      expect(wrapper.text()).toEqual('item1,item2 ');
+    });
+  });
```

Comment:
NOTE: Added a few unit tests, but visually this component has issues and I discussed it with @spong. For now, I kept these two tests to ensure that they were still getting the comma but wrote a ticket for the visual defects:
https://github.com/elastic/ingest-dev/issues/474